python/cudf/cudf/core/frame.py

```diff
@@ -292,7 +292,10 @@ def _repeat(self, count):
         result._copy_categories(self)
         return result
 
-    def _fill(self, fill_values, begin=0, end=-1, inplace=False):
+    def fill(self, fill_values, begin=0, end=-1, inplace=False):
```

Comment:
This would then be a public facing function, but the typical way to fill in Python / Pandas is via `__setitem__`. Suggest just leaving this as `_fill`.

Reply:
It sounds like you're suggesting we make `fill` private, and use `__setitem__` with a `slice` to fill a range. If that's the case, it seems like `inplace` is no longer necessary?

Reply:
`inplace` is still necessary, because there's possible situations where you'd want to fill a numerical column and have it return a newly allocated column instead of modifying your existing memory.